index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <title>Dani's FullStack Web Developer Portfolio</title>
+</head>
+<body>
+  <header class="app-bar">
+    <div class="status-bar"></div>
+    <div class="toolbar">
+      <div class="log-placeholder">
+        Welcome
+      </div>
+      <div class="hamburger-icon">
+        <img src="images/icons/hamburger-icon.svg" alt="hamburger-icon">
+      </div>
+    </div>
+  </header>
+  <main class="headline">
+    <div class="primary">
+      <h2>Hey there. I’m Dani</h2>
+        <h2 class="i-am-a">I’m a software developer</h2>
+    </div>
+    <div class="supporting-text">
+      <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+        don’t hestiate to contact me.</p>
+    </div>
+    <div class="social-media-icons">
+      <a href="https://twitter.com/danifromecuador"><img src="images/icons/twitter.svg" alt="twitter icon"></a>
+      <a href="https://linkedin.com/in/danifromecuador/"><img src="images/icons/linkedin.svg" alt="linkedin icon"></a>
+      <a href="https://medium.com/"><img src="images/icons/medium.svg" alt="medium icon"></a>
+      <a href="https://github.com/danifromecuador"><img src="images/icons/github.svg" alt="github icon"></a>
+      <a href="https://www.angellist.com/"><img src="images/icons/angellist.svg" alt="angellist icon"></a>
+    </div>
```

Comment:
Kindly update this element to use a `<ul>` tag instead of a `<div>` tag. This is for semantic reasons as ul tags are used to wrap most lists in a webpage

- In addition to that, please wrap each img in a `<li>` tag and make sure that the images are all links using an `<a>` tag. Your final code should look similar to something like this
`<ul>`
     `<li><a><img/></a></li>`
     `<li><a><img/></a></li>`
     `<li><a><img/></a></li>`
`</ul>`